src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
```suggestion
const promise1 = new Promise(resolve => {
```

Reply:
You can live just resolve if you don't use reject

Reply:
Got it, thanks.